tests/python/test_serial_execution.py

```diff
@@ -0,0 +1,37 @@
+import taichi as ti
+
+
+@ti.test(arch=ti.cpu, cpu_max_num_threads=1)
```

Comment:
Actually we can `arch=[ti.cpu, ti.cc]`?

Reply:
IIRC `ti.cc` is always serial so there is no need to test it here?